src/os_posix/os_fs.c

```diff
@@ -1062,12 +1090,12 @@ __wt_map_file(WT_FILE_HANDLE *file_handle, WT_SESSION *wt_session)
  */
 
 /*
- * __wt_prepare_remap_resize_file --
+ * __wti_prepare_remap_resize_file --
```

Comment:
Should we include "posix" in the function names that are specific to POSIX? I see we already have functions that start with "__wti_posix". I'm not sure it's necessary (and doesn't have to be done within this ticket), but it would be nice to have.

Reply:
Done